### PR TITLE
Add ParseListLoader

### PR DESCRIPTION
**Pull request by tuanchauict**
ParseListLoader is an universal loader for everykind of Adapter or something else. Just implement LoaderTarget with just a single line of code for each of three functions.

### CODE REVIEW

File: ParseListLoaderExample/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (wangmengyan95) [inline ParseListLoaderExample/.gitignore]: Can we use the root's gitignore?

File: ParseListLoaderExample/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Create an App on [Parse.com](https://parse.com) then modify strings.xml with your keys
```
Comment (wangmengyan95) [inline ParseListLoaderExample/README.md]: Nit: Follow other samples' style
```xml
    <!-- Please put your application's app ID and keys here -->
    <string name="parse_app_id">YOUR_PARSE_APP_ID</string>
    <string name="parse_client_key">YOUR_PARSE_CLIENT_KEY</string>
```

File: ParseListLoaderExample/build.gradle [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        applicationId "com.example.tuanchauict.parselistloaderexample"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```
Comment (wangmengyan95) [inline ParseListLoaderExample/build.gradle]: Nit: we do not use proguardFiles for sample project.

File: ParseListLoaderExample/build.gradle [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        }
+    }
+}
+
```
Comment (wangmengyan95) [inline ParseListLoaderExample/build.gradle]: Nit: Please follow other samples' build.gradle

File: ParseListLoaderExample/build.gradle [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+dependencies {
+    compile fileTree(include: ['*.jar'], dir: 'libs')
+    compile 'com.android.support:recyclerview-v7:21.+'
+    compile project(':ParseLoginUI')
+    compile rootProject.ext.androidSupport
+    compile files(rootProject.ext.parsePath)
+}
```
Comment (wangmengyan95) [inline ParseListLoaderExample/build.gradle]: Nit: Please follow other samples' build.gradle

File: ParseListLoaderExample/build.gradle [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(include: ['*.jar'], dir: 'libs')
```
Comment (wangmengyan95) [inline ParseListLoaderExample/build.gradle]: Do you use some local jar files?

File: ParseListLoaderExample/proguard-rules.pro [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```
Comment (wangmengyan95) [inline ParseListLoaderExample/proguard-rules.pro]: Nit: we do not use proguardFiles for sample project.

File: ParseListLoaderExample/src/androidTest/java/com/example/tuanchauict/parselistloaderexample/ApplicationTest.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
+package com.example.tuanchauict.parselistloaderexample;
```
Comment (wangmengyan95) [inline ParseListLoaderExample/src/androidTest/java/com/example/tuanchauict/parselistloaderexample/ApplicationTest.java]: Nit: Maybe delete this test?

**Comment by tuanchauict** [comment]
@wangmengyan95  : Do I need to restruct project with `src`, `res` and `AndroidManifest.xml` in the root directory?

**Comment by wangmengyan95** [comment]
Hi @tuanchauict, not quite understand your question, which root directory do you mean?

**Comment by tuanchauict** [comment]
Hi @wangmengyan95 

I mean with the default project struture of gradle, we have `java`, `res`, and `AndroidManifest.xml` in `<module_dir>/src/main` but the example from Parse, I realsized that they all are in `<module_dir>/`.

Anw, I updated the structure which was similar to Parse's examples.

**Comment by grantland** [comment]
Thanks for your contribution, but we'd suggest using `ParseQueryPager` + `RecyclerView` for this use case (#129 #130).

**Comment by tuanchauict** [comment]
Errr. Will we have Parse to use next year?

**Pull request closed without merging**